inc/saihostintf.h

```diff
@@ -230,6 +230,18 @@ typedef enum _sai_hostif_trap_id_t
     /** default packet action is forward */
     SAI_HOSTIF_TRAP_ID_MLD_V2_REPORT = 0x0000200f,
 
+    /** IP packets to local router IP address (routes with SAI_ROUTE_ATTR_NEXT_HOP_ID=SAI_SWITCH_ATTR_CPU_PORT)
```

Comment:
1. My IP should be hit in the route/host table and then should go the CPU as pkt cmd. There is no other possible logical action. Worst case some user might want to drop. 
2. Why we need specific TRAP IDs. Also do we expect the HW to do deeper packet inspection to even figure out application.